external/open62541/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ opendaq_dependency(
     NAME                open62541
     REQUIRED_VERSION    1.3.6
     GIT_REPOSITORY      https://github.com/openDAQ/open62541.git
-    GIT_REF             v1.3.6-opendaq-3
+    GIT_REF             v1.3.6-opendaq-4
```

Comment:
What about rebasing our changes on top of the latest 1.3 branch, currently 1.3.9 (plus a few extra commits)?

Reply:
That's a bit out-of-scope of this PR. I currently don't know of any features that we'd really need from the new version.